common/ownership_voucher/ownership_voucher.go

```diff
@@ -0,0 +1,92 @@
+// Package ownershipvoucher provides helper functions for generating, parsing and verifying ownership vouchers.
+package ownershipvoucher
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"go.mozilla.org/pkcs7"
+)
+
+const (
+	ovExpiry = time.Hour * 24 * 365
+)
+
+// OwnershipVoucher wraps OwnershipVoucherInner.
+type OwnershipVoucher struct {
+	OV OwnershipVoucherInner `json:"ietf-voucher:voucher"`
+}
+
+// OwnershipVoucherInner defines the Ownership Voucher format. See https://www.rfc-editor.org/rfc/rfc8366.html.
+type OwnershipVoucherInner struct {
+	CreatedOn                  string `json:"created-on"`
+	ExpiresOn                  string `json:"expires-on"`
+	SerialNumber               string `json:"serial-number"`
+	Assertion                  string `json:"assertion"`
+	PinnedDomainCert           string `json:"pinned-domain-cert"`
+	DomainCertRevocationChecks bool   `json:"domain-cert-revocation-checks"`
+}
+
+// removePemHeaders strips the PEM headers from a certificate so it can be used in an Ownership Voucher.
```

Comment:
Capitalise removePemHeaders to match function name

Reply:
Done.